<launch>
    <node pkg="swarm_vo_fuse" name="swarm_vo_visualize" type="swarm_vo_visualize.py" output="screen"> 
    </node>
    <node pkg="swarm_vo_fuse" name="test_datagen" type="test_vo_datagen.py" output="screen"> 
        <param name="drone_num" value="2" type="int" />
        <param name="self_id" value="0" type="int" />
    </node>
    <node pkg="swarm_vo_fuse" name="swarm_vo_fuse" type="swarm_vo_fuse_node" output="screen">
        <param name="max_keyframe_num" value="100" type="int" />
        <param name="thread_num" value="4" type="int" />
    </node>
      <node pkg="tf"
        type="static_transform_publisher"
        name="my_frame_link"
        args="0 0 0 0 0 0 map base_link 10" />
</launch>